modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Deleting formatted content and then press space and then digit a test places the space at the end of the content. TINY-9310
```

Comment:
I can see that the real bug was the space inserted at the wrong place, but I think we keep the changelog (and eventual release notes) much more simple and focused on what the user thinks happened.
```suggestion
- Typing after deleting formatted content could remove a space at the start of the typing. TINY-9310
```